webportal/webapp/angular.json

```diff
@@ -45,6 +45,23 @@
                   "with": "src/environments/environment.prod.ts"
                 }
               ]
+            },
+            "test": {
```

Comment:
In order to facilitate testing for this change, I added the test environment.

Reply:
Isn't that what dev is for?  Do we even have a working test cognito pool?